.travis.yml

```diff
@@ -395,6 +395,56 @@ jobs:
 env:
   global:
     TOXENV_TMPL: "'ansible${ANSIBLE_VERSION}-${TESTS_TYPE}'"
+matrix:
```

Comment:
`matrix` is an alias for `jobs`. having both may make travis config parser misbehave.

Reply:
According to the [build mentioned in the description](https://travis-ci.com/ansible/molecule/builds/100236656), travis config parser behave well.

Reply:
It's still weird to see the same section twice. Please merge them.